plugins/sudo/sudo.plugin.sh

```diff
@@ -1,11 +1,21 @@
 #! bash oh-my-bash.module
 
 function _omb_plugin_sudo_add_sudo {
+  function __omb_plugin_sudo_is_inserted {
+    [[ "${READLINE_LINE::4}" = "sudo" ]]; return $?
+  }
   if [[ -z $READLINE_LINE ]]; then
     READLINE_LINE="$(fc -ln -1 | command sed 's/^[[:space:]]\{1,\}//')"
+    ((READLINE_POINT += ${#READLINE_LINE}))
+    __omb_plugin_sudo_is_inserted && return
+  fi
+  if __omb_plugin_sudo_is_inserted; then
+    READLINE_LINE="${READLINE_LINE//sudo /}"
+    ((READLINE_POINT -= 5))
```

Comment:
This can make the cursor position negative. I think Bash handles it properly, but it is good to ensure it is non-negative.

```suggestion
    ((READLINE_POINT -= 5))
    ((READLINE_POINT >= 0)) || READLINE_POINT=0
```